_posts/en/newsletters/2021-12-22-newsletter.md

```diff
@@ -0,0 +1,665 @@
+managing [dual-funded channels][topic dual funding]---channels where
+both parties can provide some amount of the initial funding.  In
+addition to prior dual-funding work [merged][cl#4410] this year, this
+allows the party initiating the channel open to not only spend funds
+through the channel but also receive them in the channel's initial
+state.  Dual-funding is particularly useful for merchants whose primary
+use of LN is receiving payments instead of sending them.
+
+<div markdown="1" class="callout" id="releases">
+### 2021 summary<br>Major releases of popular infrastructure projects
+
+- [Eclair 0.5.0][] added support for a scalable cluster mode (see
+  [Newsletter #128][news128 akka]), block chain watchdogs ([Newsletter
+  #123][news123 watchdog]), and additional plugin hooks.
+
+- [Bitcoin Core 0.21.0][] included support for new Tor onion services
+  using [version 2 address announcement messages][topic addr v2], the
+  optional ability to serve [compact block filters][topic compact block
+  filters], and support for [signets][topic signet] (including the
+  default signet which had [taproot][topic taproot] activated).  It also
+  offered experimental support for wallets that natively use [output
+  script descriptors][topic descriptors].
+
+- [Rust Bitcoin 0.26.0][] included support for signet, version 2 address
+  announcement messages, and improvements to [PSBT][topic psbt]
+  handling.
+
+- [LND 0.12.0-beta][] included support for using [watchtowers][topic
+  watchtowers] with [anchor outputs][topic anchor outputs] and added a
+  new `psbt` wallet subcommand for working with [PSBTs][topic psbt].
+
+- [HWI 2.0.0][] contained support for multisig on the BitBox02, improved
+  documentation, and support for paying `OP_RETURN` outputs with a
+  Trezor.
+
+- [C-Lightning 0.10.0][] contained a number of enhancements to its API
+  and experimental support for [dual funding][topic dual funding].
+
+- [BTCPay Server 1.1.0][] included [Lightning Loop][news53 lightning
+  loop] support, added [WebAuthN/FIDO2][fido2 website] as a two-factor
+  authentication option, made various UI improvements, and marked a
+  switch to using [semantic versioning][semantic versioning website] for
+  version numbers moving forward.
+
+- [Eclair 0.6.0][] contained several improvements that enhanced user
+  security and privacy.  It also provided compatibility with future
+  software that may use [bech32m][topic bech32] addresses.
+
+- [LND 0.13.0-beta][] improved feerate management by making [anchor
+  outputs][topic anchor outputs] the default commitment transaction
+  format, added support for using a pruned Bitcoin full node, allowed
+  receiving and sending payments using Atomic MultiPath ([AMP][topic
+  multipath payments]), and increased LND's [PSBT][topic psbt]
+  capabilities
+
+- [Bitcoin Core 22.0][] included support for [I2P][topic anonymity
+  networks] connections, removed support for [version 2 Tor][topic
```

Comment:
Suggest removing the 'version 2 Tor' link since we have it just prior.

Reply:
I don't see the same link anywhere else. I think you may be referring to the version 2 address announcements in the Bitcoin Core 0.21.0 notes, which refer to the format how to announce node addresses, while the "version 2 Tor" here refers to Tor versions.

Reply:
@Xekyo I think @bitschmidty is referring to both the I2P and Tor links in the paragraph pointing to the same page.

@bitschmidty I'd prefer to link both of them.  I know it's a bit silly, but I don't want to play favorites, so I'm linking them both.